lib/utils/constants.dart

```diff
@@ -35,6 +35,13 @@ abstract class Constants {
   static const double borderRadius = 10.0;
   static const double dividerThickness = 2.0;
   static const String subtitleCardMail = "E-mail";
-  static const String subtitleCardPhone = "Cell phone";
+  static const String subtitleCardPhone = "Linkedin";
```

Comment:
change the name of this constant, it's not a phone anymore

Reply:
Fixed